pilot/lib/pilot/tester_manager.rb

```diff
@@ -67,7 +67,14 @@ def remove_tester(options)
         # If no groups are passed to options, remove the tester from the app-level,
         # otherwise remove the tester from the groups specified.
         if config[:groups].nil? && tester.kind_of?(Spaceship::Tunes::Tester::External)
-          test_flight_tester = Spaceship::TestFlight::Tester.find(app_id: app.apple_id, email: tester.email)
+          test_flight_testers = Spaceship::TestFlight::Tester.search(app_id: app.apple_id, text: tester.email)
```

Comment:
True, but the `search` method is at the `spaceship` level, while this is in `pilot` (and the parameter here is "email", not a more generic "test").  So, it seems cool for `pilot` to use `search` to look for text matches for the email, but `pilot` should probably assert that it is an email match before doing the removal, no?